CSharp/Library/Microsoft.Bot.Connector.Shared/JwtConfig.cs

```diff
@@ -18,6 +18,11 @@ public static class JwtConfig
         /// </summary>
         public const string ToChannelFromBotOAuthScope = "https://api.botframework.com/.default";
 
+        /// <summary>
+        /// OAuth resource URI for token acquiring
+        /// </summary>
+        public const string OAuthResourceUri = "https://api.botframework.com/";

```

Comment:
Yay. OAuth treats this as a scope and is an opaque string. Looks like ADAL treats this as a resource URI (different concept, but overloads the same slot) and I assume by the change that it's automatically applying .default as the resource path?

This is simplest if we just keep the whole, opaque URI. Possible or no?